tests/cases/fourslash/completionsSelfDeclaring2.ts

```diff
@@ -9,10 +9,12 @@
 
 verify.completions({
   marker: "1",
-  exact: completion.globalsPlus(["f1", "f2"])
+  exact: completion.globalsPlus(["f1", "f2"]),
+  isNewIdentifierLocation: true
 });
 
 verify.completions({
   marker: "2",
-  exact: ["xyz"]
+  exact: ["xyz"],
+  isNewIdentifierLocation: false
```

Comment:
This one is somewhat surprising since `T` is still generic.

Reply:
I [commented about this](https://github.com/microsoft/TypeScript/issues/42595#issuecomment-771853487) in the issue.